test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
```

Comment:
Please remove the test folder from git (with git rm...) and also ignore in git this folder, as requested in the card.